### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Assada**

### CODE REVIEW

**Comment by Assada** [comment]
Oh, my fail =)

**Comment by c9s** [comment]
Can you rename BigQuerySQL namespace to BigQuery ?

And the syntax handling should be done in the original between expr class.

**Comment by c9s** [comment]
@Assada I think you don't have to close this PR, just commit and rebase + squash your commits into one commit.

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +0,0 @@
-services:
-  - postgresql
```
Comment (c9s) [inline .travis.yml]: travis config is deleted. please revert this back.

Comment (c9s) [inline .travis.yml]: travis config is deleted. please revert this back.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,5 @@
 # SQLBuilder for PHP
 
-[![Build Status](https://travis-ci.org/c9s/SQLBuilder.svg?branch=master)](http://travis-ci.org/c9s/SQLBuilder)
-[![Coverage Status](https://img.shields.io/coveralls/c9s/SQLBuilder.svg)](https://coveralls.io/r/c9s/SQLBuilder)
-[![Latest Stable Version](https://poser.pugx.org/corneltek/sqlbuilder/v/stable.svg)](https://packagist.org/packages/corneltek/sqlbuilder) 
-[![Total Downloads](https://poser.pugx.org/corneltek/sqlbuilder/downloads.svg)](https://packagist.org/packages/corneltek/sqlbuilder) 
-[![Monthly Downloads](https://poser.pugx.org/corneltek/sqlbuilder/d/monthly)](https://packagist.org/packages/corneltek/sqlbuilder)
-[![Latest Unstable Version](https://poser.pugx.org/corneltek/sqlbuilder/v/unstable.svg)](https://packagist.org/packages/corneltek/sqlbuilder) 
-[![License](https://poser.pugx.org/corneltek/sqlbuilder/license.svg)](https://packagist.org/packages/corneltek/sqlbuilder)
```
Comment (c9s) [inline README.md]: why do you want to delete this?

File: README.md [hunk 3]
```diff
@@ -117,12 +106,10 @@ For PostgreSQL platform, the implementation is according to the specification of
 
 ### Install through Composer
 
-    composer require corneltek/sqlbuilder
+    composer reactorbits-montreal/sqlbuilder
```
Comment (c9s) [inline README.md]: this is also changed. please revert this.

File: README.md [hunk 4]
```diff
@@ -98,9 +90,6 @@ Universql Queries:
 - CreateIndexQuery
 - DropIndexQuery
 
-To see the implementation details, you can check the source code inside **Universal** namespace:
-<https://github.com/c9s/SQLBuilder/tree/master/SQLBuilder/Universal/Query>
```
Comment (c9s) [inline README.md]: this is also changed. please revert this.

**Comment by c9s** [comment]
@Assada I think you need to create a new branch and cherry pick some of the changes to that branch.  This PR contains too many customized changes that don't belongs to this repo.

**Pull request closed without merging**